### PR TITLE
[PD-1975] Added activities for NUO.

### PR DESCRIPTION
**Pull request by aspidites**
Does what it says on the tin.

### CODE REVIEW

**Comment by koonsies** [comment]
Tests failing

**Comment by aspidites** [comment]
Whoops. Thought I closed this yesterday.

**Comment by aspidites** [comment]
Should be good to go now.

File: myjobs/tests/setup.py [hunk 1]
```diff
@@ -152,6 +153,8 @@ def assertRequires(self, view_name, *activities, **kwargs):
         Asserst that the given view is only accessible when a user has a role
         with the given activities.
         """
+        if "nuo" in view_name:
+            import ipdb; ipdb.set_trace()
```
Comment (tdphillips) [inline myjobs/tests/setup.py]: debugger

Comment (aspidites) [inline myjobs/tests/setup.py]: Thanks

**Comment by aspidites** [comment]
Any ideas about cleaner way to handle activities in tests? Shoving them all in the mix works, but somewhat obfuscates what's actually required. On the other hand, examining the view should tell you this much. I don't want to address this in this specific PR, but am contemplating making another to do a tad bit of testing house cleaning.

**Comment by koonsies** [comment]
Cycling to rerun tests

**Pull request merged**